### ISSUE TITLE
[🐛 Bug]: [GRID] Some sessions remain stuck with no Owner for days blocking Slots unable to be automatically or manually deleted

### ISSUE THREAD

**Issue by KrzysztofC** [SeleniumHQ/selenium#13343]
### What happened?

After some time of daily non stop test executions, there are some Sessions that remain stuck for hours and days and have no Owner and are not removed automatically.

I tried to delete with `curl --request DELETE http://node_ip:5555/se/grid/node/session/b62d2f1e9cc22e67a36a4e655ff3d6e7 --header 'X-REGISTRATION-SECRET;'`
but that has never worked for me for Sessions with no Owner, always returns "cannot find session with id".

I also tried `curl --request DELETE http://localhost:4444/session/b62d2f1e9cc22e67a36a4e655ff3d6e7 --header 'X-REGISTRATION-SECRET;'` This has worked for me few days ago for some Sessions, but now I get the below:

{
  "value": {
    "error": "unknown command",
    "message": "Unable to find handler for (DELETE) \u002fsession\u002f1395d9965a1f6ed745a91a7a86359f5c",
    "stacktrace": ""
  }
}
Session is valid because when it's not, it responds with "invalid session id". 

![g2](https://github.com/SeleniumHQ/selenium/assets/30735798/4547b363-23bd-4f8d-bfa8-8be7ebc106b0)

![g1](https://github.com/SeleniumHQ/selenium/assets/30735798/ea8f8a57-d92c-4fce-b783-ff28359f3c5a)

### How can we reproduce the issue?

```shell
Execute many tests for days until some Sessions get stuck.
I tried to replicate but the issue never happens when executing some sample tests for prolonged time.
```

### Relevant log output

```shell
See screenshot with commands and endpoint responses.
```

### Operating System

Red Hat 8

### Selenium version

4.16.1

### What are the browser(s) and version(s) where you see this issue?

Any

### What are the browser driver(s) and version(s) where you see this issue?

Any

### Are you using Selenium Grid?

4.16.1

**Comment by KrzysztofC** [comment]
Since I already wrote a little utility for Grid maintenance, the idea was to add code to use endpoints to clear those stuck sessions, but unfortunately, as mentioned in the Issue, I'm not able to delete Sessions with no Owners.

![G3](https://github.com/SeleniumHQ/selenium/assets/30735798/e703f6b9-8198-417c-95a1-3c048d108c10)

**Comment by diemol** [comment]
We have tested that the timeout is triggered when a session is idle. I understand that you are having that situation but dumping a stacktrace and a few screenshots is not giving us much to work with. 

This is the command I use
```
curl -XDELETE --header "Content-Type: application/json; charset=utf-8" http://localhost:4444/session/e1ecfd86c2b026e034759a18d763f642
```

Have you inspected the sessions? How are you starting the Grid? If the timeout was failing we would have many reports.

**Comment by KrzysztofC** [comment]
curl -XDELETE --header "Content-Type: application/json; charset=utf-8" http://HUB_IP:PORT/session/1395d9965a1f6ed745a91a7a86359f5c
{
  "value": {
    "error": "unknown command",
    "message": "Unable to find handler for (DELETE) \u002fsession\u002f1395d9965a1f6ed745a91a7a86359f5c",
    "stacktrace": ""
  }
}

Unfortunately I don't know how to troubleshoot those Sessions.

java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar hub ${SE_OPTS}
SE_OPTS=--newsession-threadpool-size 9 --allow-cors true --reject-unsupported-caps true --session-request-timeout 25
JAVA_OPTS=-XshowSettings:vm -Xms8g -Xmx12g -Xss512k

java ${JAVA_OPTS} -jar /opt/selenium/selenium-server.jar node --config /opt/selenium/config.toml ${SE_OPTS}
JAVA_OPTS=-XshowSettings:vm -Xms1g -Xmx4g -Dselenium.LOGGER.level=WARNING -Dwebdriver.edge.driver=/opt/selenium/msedgedriver
SE_NODE_SESSION_TIMEOUT=300

One observation: If I trigger Node Drain of a Node with such stuck Session, then in some cases Node remains in Draining mode for a long time and nothing happens, while other time it would actually get rid of stuck Session and Drain successfully pretty quickly.

Maybe, since nobody else is reporting it, it's either caused by my config, OR maybe it is caused by some particular test suite code or some selenium timeout settings passed along with it.
If only that DELETE worked. Weirdly enough I did see it working few days ago, but those stuck sessions I generated myself by hammering Grid with parallel execution exceeding number of available slots.
